### PR TITLE
Growatt sensor

### PR DESCRIPTION
**Pull request by timvancann**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21004

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.growatt.markdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: growatt.png
+ha_category: Energy
+ha_release: 0.88.0
+---
+
+The Growatt sensor enables you to monitor Growatt solar panel plants that have a WiFi module that send the data to [the Growatt server](http://server.growatt.com). The sensor will pull the total `kWh` and daily `kWh` from the server.
```
Comment (arthurlutz) [inline source/_components/sensor.growatt.markdown]: Maybe https://server.growatt.com/ could be used to encourage use of https (more privacy, better security).

File: source/_components/sensor.growatt.markdown [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+title: "Growatt sensor"
+description: "Instructions on how to integrate a Growatt solar panel plant into Home Assistant."
+date: 2019-01-30 15:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: growatt.png
+ha_category: Energy
+ha_release: 0.88.0
```
Comment (arthurlutz) [inline source/_components/sensor.growatt.markdown]: Maybe add : 

```
:ha_iot_class: "Cloud Polling"
```

**Review by arthurlutz — CHANGES_REQUESTED** [review]

File: source/_components/sensor.growatt.markdown (hunk 1 above)
Comment (timvancann) [reply source/_components/sensor.growatt.markdown]: Good point, it actually is `https` 😄

File: source/_components/sensor.growatt.markdown (hunk 2 above)
Comment (timvancann) [reply source/_components/sensor.growatt.markdown]: 👍 , what does it do though?

Comment (klaasnicolaas) [reply source/_components/sensor.growatt.markdown]: https://www.home-assistant.io/blog/2016/02/12/classifying-the-internet-of-things/

File: source/_components/sensor.growatt.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+layout: page
+title: "Growatt sensor"
+description: "Instructions on how to integrate a Growatt solar panel plant into Home Assistant."
+date: 2019-01-30 15:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: growatt.png
+ha_category: Energy
```
Comment (klaasnicolaas) [inline source/_components/sensor.growatt.markdown]:
```suggestion
ha_category: 
  - Energy
  - Sensor
```

File: source/_components/sensor.growatt.markdown [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+ha_iot_class: "Cloud Polling"
+ha_release: 0.89.0
```
Comment (klaasnicolaas) [inline source/_components/sensor.growatt.markdown]:
```suggestion
ha_release: 0.89
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
After this I will approve this PR and label it with `awaits-parent`

File: source/_components/sensor.growatt.markdown (hunk 3 above)
Comment (timvancann) [reply source/_components/sensor.growatt.markdown]: Seems like CI can't handle this. What is your suggestion for the failure?

**Comment by timvancann** [comment]
@klaasnicolaas I've reverted you're suggested change as CI did not succeed. Also couldn't find any component that has multiple `ha_category`s which makes me think it's not supported.

**Comment by klaasnicolaas** [comment]
Will try it again 😉 
There are a lot of components who also have this so it is certainly supported, as [example](https://raw.githubusercontent.com/home-assistant/home-assistant.io/next/source/_components/smartthings.markdown)

The build fails again but please do not change it now, other people will take a look on it.

**Comment by klaasnicolaas** [comment]
You need to rebase your PR, I think your branch is to old.

**Comment by timvancann** [comment]
@klaasnicolaas You were right, I had indeed neglected to rebase for a while now. It all seems good now. Thank you very much!

**Comment by timvancann** [comment]
Ouch, something did go wrong while rebasing though... I just noticed this branch now has a massive amount of changes...

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
Great! 

✅ This PR can be merged as soon the parent PR is merged.

File: source/_components/sensor.growatt.markdown [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+date: 2019-01-30 15:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: growatt.png
+ha_category:
+  - Energy
+  - Sensor
+ha_iot_class: "Cloud Polling"
```
Comment (klaasnicolaas) [inline source/_components/sensor.growatt.markdown:14]:
```suggestion
ha_iot_class: Cloud Polling
```

File: source/_components/sensor.growatt.markdown [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+ha_release: "0.93"
```
Comment (klaasnicolaas) [inline source/_components/sensor.growatt.markdown:15]:
```suggestion
ha_release: 0.95
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing PR since the parent PR has been closed.

**Pull request closed without merging**